### PR TITLE
Modify deadline text to disclose immediately after fix is released.

### LINKED ISSUE #5255: Reduce disclosure policy window from 30 days after fix.
With the release of [OSV](https://github.com/google/osv), we've created a better way for users of open source (and OSS-Fuzz projects) to be aware of vulnerabilities affecting their version of a library. 

Unfortunately, the current 30 day disclosure window after a patch is released means that affected users of a vulnerable library do not get notified for up to 30 days. We are planning to reduce this to **immediately** after the fix is upstreamed. The existence of a patch in the upstream repo already discloses the vulnerability, so this is minimal additional risk in return for a large benefit (more timely notifications).

The new deadline text will read (the "30 days" part is removed):

> After notifying project authors, we will open reported issues to the public in 90 days, or after the fix is released (whichever comes earlier).

### PR DESCRIPTION
**Pull request by oliverchang**
Fixes #5255

### CODE REVIEW

**Review by inferno-chromium — APPROVED** [review]

File: docs/getting-started/bug_disclosure_guidelines.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ Following [Google's standard disclosure policy](https://googleprojectzero.blogsp
 OSS-Fuzz will adhere to following disclosure principles:
 
   - **Deadline**. After notifying project authors, we will open reported
-    issues to the public in 90 days, or 30 days after the fix is released
-    (whichever comes earlier).
+    issues to the public in 90 days, or after the fix is released (whichever
```
Comment (jonathanmetzman) [inline docs/getting-started/bug_disclosure_guidelines.md:15]: nit: maybe say "immediately after the fix is released"
Current wording is ambiguous.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

File: docs/getting-started/bug_disclosure_guidelines.md (hunk 1 above)
Comment (oliverchang) [reply docs/getting-started/bug_disclosure_guidelines.md:15]: "Immediately" is also inaccurate (we can be delayed by a day or so), so keeping the current wording.

**Comment by rlohning** [comment]
So far only the documentation changed, right? Will you send an announcement when the grace period actually changes?

**Pull request merged**